pallets/rc-migrator/src/staking/nom_pools.rs

```diff
@@ -328,3 +334,284 @@ impl<T: pallet_nomination_pools::Config> NomPoolsMigrator<T> {
 		LastPoolId::<T>::put(values.last_pool_id);
 	}
 }
+
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, DebugNoBound, PartialEq, Clone)]
+pub struct GenericCommission<AccountId: Debug, BlockNumber: Debug> {
```

Comment:
these types and imports probably causing unused import warnings. you can just create sub modules under cfg feature std here, and move everything under it.

Reply:
Fixed, thanks